src/foam/nanos/column/CommonColumnHandler.js

```diff
@@ -91,6 +91,14 @@ foam.CLASS({
       return  prop ? prop.property[property] : of.getAxiomByName(this.returnPropertyNamesForColumn(colObj))[property];
     },
     function groupObjectsThatAreRelatedToNestedProperties(of, arrayOfNestedPropertiesName, arrayOfValues) {
+      //this function creates obj for nested properties and adds it to the map
```

Comment:
I know it's a minor point, but could you please leave a space after //'s. Thx

Reply:
sure! fixed. thanks